### PR TITLE
avoid batch files

### PR DESCRIPTION
**Pull request by jsoref**
@rcruz @kflores772 here's a draft which at least seems to work on OS X (I'll test w/ Windows tomorrow).

### CODE REVIEW

**Comment by rcruz** [comment]
To clarify, this change skips the "webworks" command and directly runs node with a particular file? While this may run the same files in node, the action is no longer exactly equivalent to running a "webworks" command on terminal. This would mean we still have a problem with stdout/stderr on the "webworks" command, but we are avoiding it by implementing a workaround into the GUI. @bryanhiggins @kwallis Is this the approach we want to take?

**Comment by bryanhiggins** [comment]
AFAIK this does not affect the console only the values returned in the child process callback

File: routes/create.js [hunk 1]
```diff
@@ -30,7 +31,7 @@ module.exports = {
             execStr,
             child;
 
-            execStr = util.format('"%s" %s "%s"', cmdPath, cmd, location, projectId, name); //quoted placeholders are not defined for projectId and name as they are optional
+            execStr = util.format('"%s" "%s" %s %s', apiUtil.getNode(), apiUtil.getWebworksCli(), cmd, location, projectId, name); //quoted placeholders are not defined for projectId and name as they are optional
```
Comment (rcruz) [inline routes/create.js]: Need to wrap "location" in quotation marks. Cannot create projects with spaces in the path

File: test/unit/create.spec.js [hunk 2]
```diff
@@ -45,8 +45,8 @@ describe("create", function () {
         }, mockResponse);
 
         expect(cp.exec).toHaveBeenCalled();
-        expect(cp.exec.mostRecentCall.args[0]).toMatch(/webworks"\screate\s/);
-        expect(cp.exec.mostRecentCall.args[0]).toMatch(/hellow1"\s"projectId"\s"appname"/);
+        expect(cp.exec.mostRecentCall.args[0]).toMatch(/webworks"?\screate\s/);
+        expect(cp.exec.mostRecentCall.args[0]).toMatch(/hellow1"?\s"projectId"\s"appname"/);
```
Comment (rcruz) [inline test/unit/create.spec.js]: Why is this test changing to make the quotes optional?

**Comment by rcruz** [comment]
Quick test shows this change works on WinXP

**Comment by rcruz** [comment]
r+, merged

**Pull request closed without merging**